EdvClient.js

```diff
@@ -1089,7 +1089,14 @@ export class EdvClient {
     ]);
     delete encrypted.content;
     delete encrypted.meta;
-    delete encrypted.stream;
+
+    if(stream) {
+      encrypted.stream = {
+        sequence: encrypted.sequence,
+        chunks: (stream || {}).chunks
+      };
```

Comment:
I think this should just be the subset of things we want to keep in `encrypted.stream`:

```suggestion
      encrypted.stream = {
        sequence: encrypted.stream.sequence,
        chunks: encrypted.stream.chunks
      };
```

Reply:
Changed.